lib/postcssPlugin.js

```diff
@@ -3,16 +3,24 @@
 const _ = require("lodash")
 const createStylelint = require("./createStylelint")
 const postcss = require("postcss")
+const path = require("path")
 
-module.exports = postcss.plugin("stylelint", function () {
-  const options = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {}
-
-  const tailoredOptions/*: Object*/ = options.rules ? { config: options } : options
+module.exports = postcss.plugin("stylelint", function (options) {
+  options = options || {}
 
+  const tailoredOptions/*: Object*/ = options.rules
+    ? { config: options }
+    : options
   const stylelint = createStylelint(tailoredOptions)
+
   return (root, result) => {
+    let filePath = options.from || _.get(root, "source.input.file")
+    if (filePath !== undefined && !path.isAbsolute(filePath)) {
```

Comment:
Is this pattern the same here and in `standalone.js`? If so, is it possible to create a util named something like `absolutizePath()` and use that?